account_invoice_show_currency_rate/models/account_move.py

```diff
@@ -34,17 +34,26 @@ def _compute_currency_rate_amount(self):
         self.currency_rate_amount = 1
         for item in self.filtered("show_currency_rate_amount"):
             lines = item.line_ids.filtered(lambda x: abs(x.amount_currency) > 0)
+            inverse = item.company_id.invoice_rate_display_type != "normal"
```

Comment:
I prefer this style. It could be more "correct" to use the company of the invoice, but using the env company should be sufficient, IMO.

```suggestion
        inverse = self.env.company.invoice_rate_display_type == "inverse_rate"
        for item in self.filtered("show_currency_rate_amount"):
            lines = item.line_ids.filtered(lambda x: abs(x.amount_currency) > 0)
```